spec/models/tag_group_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe TagGroup, type: :model do
+  let(:project) { build :project, :with_tag}
+  let(:team) { build :team}
+
+  context "A project does have a tag_group" do
+    it { expect(project.tag_group.team_id).to eq(6) }
```

Comment:
We should not test id's on the specs, it will not be always the same in every computer you run the spec. Instead test by the team itself. `expect(project.tag_group.team).to eq team)`